README.md

```diff
@@ -7,7 +7,11 @@ This project contains the source code for the yiiframework.com Website.
 ## INSTALLATION
 
 Before you start, make sure you have installed [composer](https://getcomposer.org/) and [Node.js](http://nodejs.org/).
- 
+
+You also need to install [Ruby](https://www.ruby-lang.org/en/) and [Sass](http://sass-lang.com/) because the current tool-chain uses Gulp-ruby-sass. Both because it is faster than node-sass, but also because node-sass tends to lag behind Ruby sass.
```

Comment:
I am working on making it work using plain gulp-sass (node sass)

2015-11-23 0:06 GMT+01:00 Alexander Makarov <notifications@github.com>:

> In README.md
> <https://github.com/yiisoft-contrib/yiiframework.com/pull/57#discussion_r45566396>
> :
>
> > @@ -7,7 +7,11 @@ This project contains the source code for the yiiframework.com Website.
> >  ## INSTALLATION
> >
> >  Before you start, make sure you have installed [composer](https://getcomposer.org/) and [Node.js](http://nodejs.org/).
> > -
> > +
> > +You also need to install [Ruby](https://www.ruby-lang.org/en/) and [Sass](http://sass-lang.com/) because the current tool-chain uses Gulp-ruby-sass. Both because it is faster than node-sass, but also because node-sass tends to lag behind Ruby sass.
>
> Not depending on Ruby, I think, is good. I have it on my computer but
> still...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yiisoft-contrib/yiiframework.com/pull/57/files#r45566396>
> .
>